app/models/landable/traffic/attribution.rb

```diff
@@ -28,6 +28,8 @@ def transform(parameters)
 
         def lookup(parameters)
           where(transform(parameters)).first_or_create
+        rescue ActiveRecord::RecordNotUnique
+          retry
```

Comment:
Could this cause an infinite loop? If the record isn't unique, why would retrying ever succeed?